### DISCUSSION TITLE
Gif support

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#256] [Ideas]
Pls add sending gif support I love the app but I wish I could send gifs

**Comment by Aga-C** [comment]
This is general discussion. Please create new discussions, feature requests or bugs in the repository of the app you mean.

**Reply by ghost** [reply]
How do I comment on Fossify messages

**Reply by Aga-C** [reply]
Here's a repository for Fossify Messages: https://github.com/FossifyOrg/Messages. You can either create a new discussion, or report a new feature request in Issues.
